.expeditor/scripts/omnibus_chef_build.ps1

```diff
@@ -0,0 +1,66 @@
+$ScriptDir = Split-Path -Path $MyInvocation.MyCommand.Definition -Parent
+
+if ($env:BUILDKITE_ORGANIZATION_SLUG -eq "chef-oss" )
+{
+  Write-Output "--- Generating self-signed Windows package signing certificate"
+  $thumb = (New-SelfSignedCertificate -Type Custom -Subject "CN=Chef Software, O=Progress, C=US" -KeyUsage DigitalSignature -FriendlyName "Chef Software Inc." -CertStoreLocation "Cert:\LocalMachine\My" -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.3", "2.5.29.19={text}")).Thumbprint 
+}
+else
+{
+  Write-Output "--- Installing Windows package signing certificate"
+  $windows_certificate_json = "windows-package-signing-certificate.json"
+  $windows_certificate_pfx = "windows-package-signing-certificate.pfx"
+  
+  aws ssm get-parameter --name "windows-package-signing-cert" --with-decryption --region "us-west-1" --query Parameter.Value --output text | Set-Content -Path $windows_certificate_json
+  If ($lastexitcode -ne 0) { Throw $lastexitcode }
+  
+  $cert_passphrase = Get-Content $windows_certificate_json | ConvertFrom-Json | Select-Object -ExpandProperty cert_passphrase | ConvertTo-SecureString -asplaintext -force
+  Get-Content $windows_certificate_json | ConvertFrom-Json | Select-Object -ExpandProperty cert_content_base64 | Set-Content -Path $windows_certificate_pfx
+  Remove-Item -Force $windows_certificate_json
+  Import-PfxCertificate $windows_certificate_pfx -CertStoreLocation Cert:\LocalMachine\My -Password $cert_passphrase
+  Remove-Item -Force $windows_certificate_pfx
+  $thumb = "13B510D1CF1B3467856A064F1BEA12D0884D2528"
+}
+
+Write-Output "THUMB=$thumb"
+
+$env:ARTIFACTORY_BASE_PATH="com/getchef"
+$env:ARTIFACTORY_ENDPOINT="https://artifactory-internal.ps.chef.co/artifactory"
+$env:ARTIFACTORY_USERNAME="buildkite"
+
+Write-Output "--- Install Chef Foundation"
+. { iwr -useb https://omnitruck.chef.io/chef/install.ps1 } | iex; install -channel "current" -project "chef-foundation" -v $CHEF_FOUNDATION_VERSION
```

Comment:
Would prefer `Invoke-WebRequest` and `Invoke-Expression` in script. GitHub doesn't appear to recognize standard aliases and it helps delineate standard PowerShell from ruby or other toolchain invocations.

Reply:
Done